components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -282,6 +282,28 @@ - (void)testCustomElevation {
   XCTAssertEqualWithAccuracy(self.bottomNavBar.elevation, customElevation, 0.001);
 }
 
+- (void)testDefaultShadowColor {
+  // Then
+  XCTAssertEqualObjects(self.bottomNavBar.shadowColor, UIColor.blackColor);
+  XCTAssertEqual(self.bottomNavBar.layer.shadowColor, UIColor.blackColor.CGColor,
+                 @"(%@) is not equal to (%@)", self.bottomNavBar.layer.shadowColor,
+                 UIColor.blackColor.CGColor);
+}
+
+- (void)testCustomShadowColor {
+  // Given
+  UIColor *fakeColor = UIColor.orangeColor;
+
+  // When
+  self.bottomNavBar.shadowColor = fakeColor;
+
+  // Then
+  XCTAssertEqualObjects(self.bottomNavBar.shadowColor, fakeColor);
+  XCTAssertEqual(self.bottomNavBar.layer.shadowColor, fakeColor.CGColor,
```

Comment:
Same as comment for method above - `CGColor` comparison should use either `UIColor` conversion or `CGColorEqualToColor()`.

Reply:
Updated to use CGColorEqualToColor.